.github/workflows/ci.yml

```diff
@@ -52,14 +52,14 @@ jobs:
     steps:
       - uses: actions/setup-node@v4
         with:
-          node-version: '18'
+          node-version: '20'
```

Comment:
test run fine so lets run on 20.

Reply:
you may ask why not higher? Well this is free (no eng time was spent) whereas there are warnings on 22 at the minimum. we can do that upgrade later